frontend/helpers/generateFakeVault.ts

```diff
@@ -0,0 +1,132 @@
+import {
+    LiquidationLimits,
+    OraclePrices,
+    Vault,
+    VaultAmount,
+    VaultBase,
+    VaultTransaction,
+    VaultTransactionFees,
+} from 'auctions-core/src/types';
+import faker from 'faker';
+import COLLATERALS from 'auctions-core/src/constants/COLLATERALS';
+import BigNumber from 'bignumber.js';
+import { random } from 'lodash';
+
+const generateFakeVaultBase = function (): VaultBase {
+    const id = faker.datatype.number();
+    const address = faker.finance.ethereumAddress();
+    const collateralType = faker.helpers.randomize(Object.keys(COLLATERALS));
+    const network = 'mainnet';
+    const lastSyncedAt = faker.date.recent();
+
+    return {
+        id,
+        address,
+        collateralType,
+        network,
+        lastSyncedAt,
+    };
+};
+
+const generateFakeVaultAmount = function (): VaultAmount {
+    const initialDebtDai = new BigNumber(faker.finance.amount());
+    const collateralAmount = new BigNumber(faker.finance.amount());
+
+    return {
+        initialDebtDai,
+        collateralAmount,
+    };
+};
+
+const generateFakerLiquidationLimits = function (): LiquidationLimits {
+    const maximumProtocolDebtDai = new BigNumber(faker.finance.amount());
+    const currentProtocolDebtDai = maximumProtocolDebtDai.dividedBy(2);
```

Comment:
could we divide here and below by some natural number that is not higher than 5? Otherwise it seems like too convenient number that might cause us to miss some failing case